pandas/tests/frame/methods/test_to_numpy.py

```diff
@@ -23,11 +23,15 @@ def test_to_numpy_dtype(self):
         tm.assert_numpy_array_equal(result, expected)
 
     @td.skip_array_manager_invalid_test
-    def test_to_numpy_copy(self):
+    def test_to_numpy_copy(self, using_copy_on_write):
         arr = np.random.randn(4, 3)
         df = DataFrame(arr)
-        assert df.values.base is arr
-        assert df.to_numpy(copy=False).base is arr
+        if using_copy_on_write:
+            assert df.values.base is not arr
+            assert df.to_numpy(copy=False).base is not arr
```

Comment:
Can you add something like `df.to_numpy(copy=False).base is df.values.base` (because I think this was part of the intention of the test to verify that `to_numpy(copy=False)` didn't make a copy, and not so much that `DataFrame(arr)` doesn't make a copy)

Reply:
Done